### PR TITLE
Feature/fixes sharing and activity

### PR DESCRIPTION
**Pull request by norbertschuler**
Fixes #

Changes in this pull request:
- added more spacing to footer links
- added German translations (for better testing)
- shortened one translation (breaking otherwise)
- caught iOS cancellation of sharing to avoid showing error message
- avoided not showing import dialog by hidden alert after picker is done

@ankitecd please test this on Android

### CODE REVIEW

File: app/components/AdditionalData/AdditionalDataSettings.tsx [hunk 1]
```diff
@@ -47,16 +47,18 @@ const AdditionalDataSettings = () => {
     };
 
     Share.open(options).catch((err) => {
-      setAlertHeading(i18next.t('label.something_went_wrong'));
-      setAlertMessage(i18next.t('label.share_additional_data_error'));
-      setShowSecondaryButton(false);
-      setShowAlert(true);
+      if (err?.error?.code != 'ECANCELLED500') { // iOS cancel button pressed
```
Comment (norbertschuler) [inline app/components/AdditionalData/AdditionalDataSettings.tsx:50]: I am not sure if we can rely on this code for future version of iOS, but this is what my iOS simulator returns when I cancel the sharing dialog.

File: app/languages/de/user/editUserProfilelabels.json [hunk 2]
```diff
@@ -11,12 +11,12 @@
   "manage_offline": "Offlinekarten verwalten",
   "manage_species": "Arten verwalten",
   "activity_logging": "Aktivität aufzeichnen",
-  "activity_logs": "Aktivitätsaufzeichungen",
+  "activity_logs": "Aktivitäten",
```
Comment (norbertschuler) [inline app/languages/de/user/editUserProfilelabels.json:14]: This translation is too long now with the sharing icon behind and got broken into two lines, so I choose a shorter translation. Other language or smaller display might still have this problem.

File: app/components/AdditionalData/AdditionalDataSettings.tsx [hunk 3]
```diff
@@ -82,11 +84,11 @@ const AdditionalDataSettings = () => {
 
   const importJsonFile = async () => {
     try {
-      setShowAlert(false);
       // pick json file from file system
       const res = await DocumentPicker.pick({
         type: [DocumentPicker.types.allFiles],
       });
+      setShowAlert(false);
```
Comment (norbertschuler) [inline app/components/AdditionalData/AdditionalDataSettings.tsx:91]: On my iOS simulator this has hidden the picker dialog.

**Review by ankitecd — APPROVED** [review]

File: app/components/AdditionalData/AdditionalDataSettings.tsx (hunk 1 above)
Comment (norbertschuler) [reply app/components/AdditionalData/AdditionalDataSettings.tsx:50]: I could also successfully test this on iOS 14.6 with my iPhone X.

**Pull request merged**